docs/source/getting_started/index.rst

```diff
@@ -90,58 +78,44 @@ In order to get those files, you need to clone the
   save some time and bandwidth. If you want to get a full commit history, you
   can omit this option.
 
+We should adjust Iroha config to match Postgres host it will connect to. Edit ``host`` parameter of ``config.sample`` file in ``./iroha/example`` directory. Set it to ``some-postgres``.
+
+.. Attention:: In the usual situation, you need to provide a config file, generate
+  genesis block and keypair. However, as a part of this guide, we provide an
+  example configuration for you. Please do not use these settings in a
+  production.
+
 Starting Iroha Container
 ^^^^^^^^^^^^^^^^^^^^^^^^
 We are ready to launch our Iroha container. Let's do it with the following
 command
 
 .. code-block:: shell
 
-  docker run -it --name iroha \
+  docker run -d --name iroha \
   -p 50051:50051 \
   -v $(pwd)/iroha/example:/opt/iroha_data \
   -v blockstore:/tmp/block_store \
+  -e KEY=node0 \
   --network=iroha-network \
-  --entrypoint=/bin/bash \
   hyperledger/iroha:develop
 
 Let's look in detail what this command does:
 
-- ``docker run -it --name iroha \`` attaches you to docker container called
-  ``iroha``
+- ``docker run -d --name iroha \`` instructs Docker to run container called
+  ``iroha`` in detached state.
+- ``-p 50051:50051 \`` maps container's port 50051 to the corresponding one on the host.
 - with ``$(pwd)/iroha/example:/opt/iroha_data \`` we add a folder containing
   our prepared configuration to a docker container into ``/opt/iroha_data``.
-- ``-v blockstore:/tmp/block_store \`` adds a persistent block storage which
-  we created before to a container, so our blocks won't be lost after we stop
-  the container
+- ``-v blockstore:/tmp/block_store \`` adds a persistent block storage to a container, so our blocks won't be lost after we stop the container.
+- ``-e KEY=node0 \`` instructs Iroha to use keys from ``example`` directory (``node0.priv`` and ``node0.pub``).
 - ``--network=iroha-network \`` adds our container to previously created
-  ``iroha-network``, so Iroha and Postgres could see each other.
-- ``--entrypoint=/bin/bash \`` Because ``hyperledger/iroha`` has
-  the custom script which runs after starting the container, we want to
-  override it so we can start Iroha Daemon manually.
-- ``hyperledger/iroha:develop`` is the image which has the ``develop``
-  branch.
-
-Launching Iroha Daemon
```

Comment:
To be honest, when I was writing this dock NOT using an entrypoint.sh and entering `irohad` command was made on purpose, so newcomers have a chance to understand what's going on in the container (no autostarting everything), so I want people to suffer :D

Reply:
So I am not sure if we really need to remove this part, but I guess it will be easier for people (if fe adjust our image)